synfig-studio/src/synfigapp/vectorizer/global_variables.h

```diff
@@ -0,0 +1,42 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file global _variable.h
+**	\brief Template File
+**
+**	$Id$
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
+**	Copyright (c) 2007 Chris Moore
+**
+**	This package is free software; you can redistribute it and/or
+**	modify it under the terms of the GNU General Public License as
+**	published by the Free Software Foundation; either version 2 of
+**	the License, or (at your option) any later version.
+**
+**	This package is distributed in the hope that it will be useful,
+**	but WITHOUT ANY WARRANTY; without even the implied warranty of
+**	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+**	General Public License for more details.
+**	\endlegal
+*/
+/* ========================================================================= */
+
+/* === S T A R T =========================================================== */
+
+#ifndef __SYNFIG_APP_GLOBAL_VARIABLE_H
+#define __SYNFIG_APP_GLOBAL_VARIABL_H
```

Comment:
Missing `E` in the word `VARIABLE`.
I also suggest renaming it to `VARIABLES` so that it matches the file name.